src/components/Dropdown/DropdownMenuItem/DropdownMenuItem.module.css

```diff
@@ -0,0 +1,92 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/**
+ * Menu items
+ */
+.item {
+    position: relative;
+    display: flex;
+    box-sizing: border-box;
+    align-items: center;
+    border-radius: var(--f-corner-4px);
+    color: var(--f-black-80);
+    cursor: pointer;
+    font-family: var(--f-font-standard);
+    font-size: var(--f-font-14-size);
+    gap: var(--f-spacing-2x);
+    transition: background-color var(--f-transition-ease-in-out), color var(--f-transition-ease-in-out);
+}
+
+.item.small {
+    min-height: 36px;
+    padding: var(--f-spacing-2x) var(--f-spacing-5x);
+}
+
+.item.large {
+    min-height: 62px;
+    padding: var(--f-spacing-3x) var(--f-spacing-5x);
+}
+
+.item.active {
+    color: var(--f-black-100);
+    font-weight: 500;
+}
+
+.item:hover {
+    background-color: var(--f-black-00);
+    color: var(--f-black-100);
+}
+
+.item:focus {
+    background-color: var(--f-white);
+    box-shadow: var(--f-shadow-element-focused-background-white);
+}
+
+.item svg {
+    flex-shrink: 0;
+}
+
+.content {
+    flex: 1;
+    padding-right: var(--f-spacing-1x);
```

Comment:
```suggestion
```

I don't think it is needed as we have the `grid-gap` set and I don't think it affects anything